care/facility/tests/test_patient_consultation_api.py

```diff
@@ -8,28 +8,28 @@ class TestPatientConsultationApi(TestBase):
     def get_base_url(self):
         return "/api/v1/consultation"
 
-    def get_list_representation(self, obj, facility_obj=None) -> dict:
+    def get_list_representation_facility(self, facility):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5394093)